chart/templates/rbac.yaml

```diff
@@ -3,93 +3,30 @@ kind: ClusterRole
 metadata:
   name: {{ .Release.Name }}
 rules:
-- apiGroups:
-    - management.cattle.io
-  resources:
-    - 'settings'
-  verbs:
-    - 'get'
-    - 'watch'
-    - 'list'
-- apiGroups:
-  - ""
-  resources:
-  - 'events'
-  verbs:
-  - 'patch'
-  - 'create'
-- apiGroups:
-  - ""
-  resources:
-  - 'secrets'
-  verbs:
-  - 'get'
-  - 'watch'
-  - 'list'
-  - 'create'
-  - 'update'
-  - 'patch'
-  - 'delete'
-- apiGroups:
-  - ""
-  resources:
-  - 'pods'
-  - 'serviceaccounts'
-  verbs:
-  - 'get'
-  - 'create'
-  - 'delete'
-  - 'list'
-  - 'watch'
-- apiGroups:
-    - "rbac.authorization.k8s.io"
-  resources:
-    - 'rolebindings'
-    - 'roles'
-  verbs:
-    - 'get'
-    - 'create'
-    - 'delete'
-    - 'watch'
-- apiGroups:
-  - ""
-  resources:
-  - 'pods/log'
-  verbs:
-  - 'get'
-- apiGroups:
-  - elemental.cattle.io
-  resources:
-  - '*'
-  verbs:
-  - '*'
-- apiGroups:
-  - fleet.cattle.io
-  resources:
-  - 'bundles'
-  verbs:
-  - '*'
-- apiGroups:
-  - apiextensions.k8s.io
-  resources:
-  - customresourcedefinitions
-  verbs:
-  - '*'
-- apiGroups:
-  - apiextensions.k8s.io
-  resources:
-  - customresourcedefinitions
-  verbs:
-  - 'list'
-  - 'get'
-- apiGroups:
-    - cluster.x-k8s.io
-  resources:
-    - machines
-  verbs:
-    - 'list'
-    - 'get'
-    - 'watch'
+  - apiGroups: ['']
+    resources: ['secrets']
+    verbs: ['get', 'list', 'create', 'watch', 'update']
+  - apiGroups: ['']
+    resources: ['pods', 'serviceaccounts']
+    verbs: ['get', 'list', 'watch']
```

Comment:
I think the operator needs to create serviceaccounts in the fleet-default namespace.   In particular it should create one for each MachineRegistration.

Reply:
Thank you, added `serviceaccounts` create permission